xcube/server/testing.py

```diff
@@ -65,6 +66,14 @@ def setUp(self) -> None:
 
         self.http = urllib3.PoolManager()
 
+        while True:
+            # noinspection PyBroadException
+            try:
+                self.fetch('/')
+                break
+            except Exception:
```

Comment:
This will loop forever if for some (other) reasons `fetch` raises.
Why this broad exception type? Isn't there a more concrete type when running into the race condition?

Reply:
E.g.  `MaxRetryError` and others.

Reply:
Thanks, adressed this.